docs/triggerbindings.md

```diff
@@ -13,30 +12,11 @@ metadata:
   name: pipeline-binding
   namespace: tekton-pipelines
 spec:
-  templatebindings:
-    - templateref:
-        name: pipeline-template
-      params:
-        - name: gitrevision
-          value: $(event.head_commit.id)
-        - name: gitrepositoryurl
-          value: $(event.repository.url)
-```
-```YAML
-apiVersion: tekton.dev/v1alpha1
-kind: TriggerBinding
-metadata:
-  name: pipeline-binding
-  namespace: tekton-pipelines
-spec:
-  templatebindings:
-    - templateref:
-        name: pipeline-template
-      params:
-        - name: gitrevision
-          value: $(event.head_commit.id)
-        - name: gitrepositoryurl
-          value: $(event.repository.url)
+  outputParams:
+    - name: gitrevision
+      value: $(event.head_commit.id)
+    - name: gitrepositoryurl
+      value: $(event.repository.url)
```

Comment:
can we show `inputParams` and the other fields in this example as well, and document how they work?

Reply:
Since we don't have conditionals/header matching yet, I omitted them for the time being.

Reply:
hm id rather either:

- not include them in the types until we're ready to document them
- include them in the docs but not have the functionality (which is what we've done for most other types anyway)

my personal feeling is that before 0.1 it's okay to have our docs be aspirational - if we intend to have the functionality in 0.1, i think we can put it in the docs before it actually works (as long as it does before 0.1). but if we _dont_ intend to include it in 0.1, i dont think we should include the types in the code at all